apps/workspaces/migrations/0060_workspace_org_settings.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.27 on 2026-01-12 11:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workspaces', '0059_featureconfig_import_billable_field_for_projects'),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)